data/zipDB 2.js

```diff
@@ -10172,7 +10172,7 @@ const zipDB = {
   "94303": {
     area: "Rest_Of_California",
     county: "Santa Clara",
-    town: "Palo Alto",
+    town: "Palo Alto/East Palo Alto",
```

Comment:
I guess we can do this is there a specific reason? The database is the USPS naming conventions.

I would prefer to do "East Palo Alto", etc but this is just literally from the post office so we might want to think about a more generic solution in the future.